functions/system.bash

```diff
@@ -341,6 +344,22 @@ Finally, all common serial ports can be made accessible to the openHAB java virt
     sed -i "/^[[:space:]]*EXTRA_JAVA_OPTS/s/-Dgnu.io.rxtx.SerialPorts=[^ \"]*//g" /etc/default/openhab2
   fi
 
+  if [[ $selection == *"4"* ]]; then
+    if is_pithree || is_pithreeplus || is_pifour; then
+      cond_echo "Adding 'dtoverlay=disable-wifi' to /boot/config.txt (RPi3/4)"
```

Comment:
I like the disable-... overlays for disabling unused hardware, but this will get us into trouble somewhere else in the code. We have rfkill commans through the codebase which will break.
Enabling an interace always requires reboot.
Need to see where this causes problems.

Same for bt in case we imlement it similarly.

Reply:
I saw only one rfkill (except in wifi_setup()) and inserted a check there.
Ok?